editor/plugins/debugger_editor_plugin.cpp

```diff
@@ -190,6 +192,9 @@ void DebuggerEditorPlugin::_notification(int p_what) {
 		case NOTIFICATION_READY: {
 			_update_debug_options();
 		} break;
+		case NOTIFICATION_PROCESS: {
+			file_server->poll();
+		} break;
```

Comment:
For `_notification` switches we try to keep a line between each case as those are often big, function-like scopes. Here they're small but consistency is important :)
```suggestion
		} break;

		case NOTIFICATION_PROCESS: {
			file_server->poll();
		} break;
```